immich.json

```
@@ -0,0 +1,128 @@
+        ]
+      },
+      "immich-redis": {
+        "image": "valkey/valkey",
+        "tag": "8-bookworm",
+        "launch_order": 2,
+        "opts": [
+          [ "-e", "REDIS_HOSTNAME=immich-redis" ]
+        ]
+      },
+      "immich-machine-learning": {
+        "image": "ghcr.io/immich-app/immich-machine-learning",
+        "tag": "release",
+        "launch_order": 3,
+        "environment": {
+          "UPLOAD_LOCATION": {
+              "description": "The location where your uploaded photos/videos are stored. Has to be the same as Library Share. Note: enter the full path to the share.",
+              "label": "Upload location [e.g. /mnt2/immich-library]"
+          },
+          "DB_DATA_LOCATION": {
+              "description": "The location where your database files are stored. Has to be the same as Database Share. Note: enter the full path to the share.",
+              "label": "Database location [e.g. /mnt2/immich-db]"
+          }
+        },
+        "volumes": {
+          "/cache": {
+            "description": "Choose a Cache Share for the machine-learning models.",
+            "label": "Cache Share [e.g. immich-cache]"
+          }
+        },
+        "opts": [
+          [ "-e", "DB_USERNAME=postgres" ],
+          [ "-e", "DB_PASSWORD=postgres" ],
+          [ "-e", "DB_DATABASE_NAME=immich" ],
+          [ "-e", "DB_HOSTNAME=immich-database" ]
+        ]
+      },
+      "immich-server": {
+        "image": "ghcr.io/immich-app/immich-server",
+        "tag": "release",
+        "launch_order": 4,
+        "ports": {
+          "2283": {
+            "description": "Immich API/Server port.",
+            "label": "Web interface port [e.g. 2283]",
+            "host_default": 2283,
+            "protocol": "tcp",
+            "ui": true
+          }
+        },
+        "environment": {
+          "UPLOAD_LOCATION": {
+              "description": "The location where your uploaded photos/videos are stored. Has to be the same as Library Share. Note: enter the full path to the share.",
+              "label": "Upload location [e.g. /mnt2/immich-library]",
+              "index": 1
+          },
+          "DB_DATA_LOCATION": {
+              "description": "The location where your database files are stored. Has to be the same as Database Share. Note: enter the full path to the share.",
+              "label": "Database location [e.g. /mnt2/immich-db]",
+              "index": 2
+          },
+          "IMMICH_LOG_LEVEL": {
+              "description": "Log level (verbose, debug, log, warn, error).",
+              "label": "Log level [e.g. log]",
+              "index": 3
+          },
+          "IMMICH_IGNORE_MOUNT_CHECK_ERRORS": {
+              "description": "When Immich starts, it checks if it can read and write to the storage volume mounts. Set to true to disable checks.",
+              "label": "Ignore error checks [e.g. false]",
+              "index": 4
+          }
+        },
```

```suggestion
        "environment": {
          "IMMICH_LOG_LEVEL": {
              "description": "Log level (verbose, debug, log, warn, error).",
              "label": "Log level [e.g. log]"
          },
          "IMMICH_IGNORE_MOUNT_CHECK_ERRORS": {
              "description": "When Immich starts, it checks if it can read and write to the storage volume mounts. Set to true to disable checks.",
              "label": "Ignore error checks [e.g. false]"
          }
        },
```
I think this is how this would look like with the variables removed. I also removed the index (so it will be sorted alphabetically) for simplicity.